include/ibm/management_console_rest.hpp

```diff
@@ -642,38 +851,284 @@ inline void
+                    timeout->cancel();
+                }
+            }
+        };
+
+        std::string matchStr(
+            "interface='org.freedesktop.DBus.Properties',type='"
+            "signal',"
+            "member='PropertiesChanged',path='/xyz/openbmc_project/certs/"
+            "ca/entry/" +
+            entryId + "'");
+
+        ackMatches.emplace(
+            entryId, std::make_unique<sdbusplus::bus::match::match>(
+                         *crow::connections::systemBus, matchStr, callback));
+    },
+        "xyz.openbmc_project.Certs.ca.authority.Manager",
+        "/xyz/openbmc_project/certs/ca", "xyz.openbmc_project.Certs.Authority",
+        "SignCSR", csrString);
+}
+
+inline void
+    handlePassThrough(const crow::Request& req,
+                      const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                      const std::string& name)
+{
+    std::vector<int32_t> command;
+    if (!redfish::json_util::readJsonPatch(req, asyncResp->res, "Send",
+                                           command))
     {
-        BMCWEB_LOG_ERROR("Unsupported character in filename: {}", fileName);
-        res.jsonValue["Description"] = "Unsupported character in filename";
-        return false;
+        BMCWEB_LOG_DEBUG("Not a Valid JSON");
+        asyncResp->res.result(boost::beast::http::status::bad_request);
+        return;
     }
 
-    // Check the filename length
-    if (fileName.length() > 20)
+    if (command.empty())
     {
-        BMCWEB_LOG_ERROR("Name must be maximum 20 characters. "
-                         "Input filename length is: {}",
-                         fileName.length());
-        res.jsonValue["Description"] = "Filename must be maximum 20 characters";
-        return false;
+        BMCWEB_LOG_ERROR("Command is empty");
+        asyncResp->res.result(boost::beast::http::status::bad_request);
+        return;
     }
 
-    return true;
+    auto respHandler = [asyncResp](const boost::system::error_code ec,
+                                   const std::vector<int32_t>& resp) {
+        if (ec)
+        {
+            BMCWEB_LOG_ERROR("handlePassThrough respHandler got error, ec = {}",
+                             ec.value());
+            asyncResp->res.result(
+                boost::beast::http::status::internal_server_error);
+            return;
+        }
+
+        if (resp.empty())
+        {
+            redfish::messages::internalError(asyncResp->res);
+            return;
+        }
+
+        std::string strData = "ai " + std::to_string(resp.size());
+        for (const auto& value : resp)
+        {
+            strData.append(" ");
+            strData.append(std::to_string(value));
+        }
+
+        asyncResp->res.addHeader("Content-Type", "application/octet-stream");
+        asyncResp->res.write(strData.c_str());
+    };
+
+    crow::connections::systemBus->async_method_call(
+        respHandler, "org.open_power.OCC.Control",
```

Comment:
which feature needs this OCC pass through commands?

Reply:
https://github.com/ibm-openbmc/bmcweb/commit/3ba2a2e767acb811c9433ad73bfe217ce44bae9a